docs/question/index.md

```diff
@@ -75,7 +87,19 @@ The `info.json` file for each question defines properties of the question. For e
 | `shareSourcePublicly`    | boolean | Whether the source code of the question should be available                                                                                                            |
 | `sharingSets`            | array   | Sharing sets which the question belongs to                                                                                                                             |
 
-See the [reference for `infoQuestion.json`](../schemas/infoQuestion.md) for more information about what can be added to this file.
+!!! note "Local development note"
+
+    Any time you edit a question `info.json` file on a local copy of PrairieLearn, you need to click “Load from disk” to reload the changes. Edits to HTML or Python files can be picked up by reloading the page. You might also need to generate a new variant of a question to run new Python code.
```

Comment:
```suggestion
    Any time you edit a question `info.json` file on a local copy of PrairieLearn, you need to click “Load from disk” to reload the changes. Although edits to HTML or Python files may be picked up by reloading the page, you might also need to generate a new variant in some cases, such as changes to question generation or to element arguments.
```

Reply:
```
Most edits to HTML or Python files will be picked up by reloading the page. However, if you made changes to question generation or element arguments, you might also need to generate a new variant.
```

workshopping...

Reply:
I applied the workshopped text in 189fcab7d634a129d31bfb5c5fd1de4e2ee3cbd9.